doc/BUILD_ON_LINUX.md

```diff
@@ -84,25 +98,36 @@ cargo make --profile development-linux-x86 flowy-sdk-dev
 cargo make --profile production-linux-x86 flowy-sdk-release
 ```
 
-11. Build app_flowy
+1. Build app_flowy
 ```shell
 # for development
 cargo make -p development-linux-x86 appflowy-linux-dev
 
 # for production, find binary from app_flowy/product/<version>/linux/<build type>/AppFlowy/
 cargo make -p production-linux-x86 appflowy-linux
+```
 
-# tips
+## Step 4: Run the application
+------------------------------
+```
+cd [frontend/]app_flowy/product/0.0.2/linux/Debug/AppFlowy/app_flowy
+./app_flowy
+```
 # run Linux GUI application through x11 on windows (use MobaXterm)
 # for instance:
 # export DISPLAY=localhost:10
-# cd app_flowy/product/0.0.2/linux/Release/AppFlowy
-# ./app_flowy
-```
 
-## [Optional] Step 3: Build Server side application (optional if you don't need to host web service locally)
+## [Optional] Step 5: Build Server side application (if you need to host web service locally)
 ------------------------------
 
+1. Environment settings
```

Comment:
I think this step is unnecessary because these YAML files only work for the backend. The AppFlowy frontend and backend are not connected by now.

Reply:
I totally agree, I just didn't want to remove anything from the doc :)